odoo/addons/base/models/ir_model.py

```diff
@@ -512,6 +513,9 @@ class IrModelFields(models.Model):
                                                       "    name, partner_id.name")
     store = fields.Boolean(string='Stored', default=True, help="Whether the value is stored in the database.")
 
+    domain_model = fields.Char(help="For Domain fields, the model on which it apply")
+    domain_model_field = fields.Char(help="For Domain fields, The field that contain the domain on which it apply.")
```

Comment:
Ahem, isn't it too late for schema changes?

Reply:
It will not be the first this happen in a `.5` version.
It would also be better to get this change in version 15.0